blue-hat/cbeer/blue_hat.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+  "flag"
+  "fmt"
+  "os"
+  "image"
+  "image/png"
+  "image/color"
+)
+
+func main() {
+  infilePtr := flag.String("infile", "red_hat.png", "Input file name")
+  outfilePtr := flag.String("outfile", "blue_hat.png", "Output file name")
+
+  flag.Parse()
+
+  fmt.Println("input file name:  ", *infilePtr)
```

Comment:
When printing comma separated strings you already get a whitespace between them.